### PR TITLE
Removed local repo lines

### PR DESCRIPTION
**Pull request by ColonelPanics**
Local repository setup is now going to be handled outside of metalware-defaults. New local repo setup will be described in the HPC Cluster Platform Knowledgebase. 

_Note: I have tested deploying a VM using this branch and didn't hit any issues regarding yum repos_

### CODE REVIEW

**Comment by bobwhitelock** [comment]
Looks reasonable to me if this is a change we want to make (which I imagine @ste78 knows more about).

One question, do we want to remove the `files/core/configs/yum.*` files themselves now since they're unused?

**Comment by ColonelPanics** [comment]
Oops, yeah the local yum conf files should be removed as well, will sort that now.

**Comment by bobwhitelock** [comment]
OK, LGTM then!

**Review by bobwhitelock — APPROVED** [review]

File: files/core/base.sh [hunk 1]
```diff
@@ -21,7 +18,7 @@ install_file authorized_keys /root/.ssh/authorized_keys
 chmod 600 /root/.ssh/authorized_keys
 echo "StrictHostKeyChecking no" >> /root/.ssh/config
 
-yum -y install yum-plugin-priorities yum-utils
+yum -y install yum-utils
```
Comment (sierra-tango-echo) [inline files/core/base.sh]: You can take the yum-utils line out as well, install it as part of your client local repo setup script

**Review by sierra-tango-echo — CHANGES_REQUESTED** [review]

File: files/core/base.sh (hunk 1 above)
Comment (ColonelPanics) [reply files/core/base.sh]: Done in commit 4d6cdf3

**Review by sierra-tango-echo — APPROVED** [review]

**Pull request merged**